engine/flink/avro-util/src/main/scala/pl/touk/nussknacker/engine/avro/schemaregistry/confluent/ConfluentUtils.scala

```diff
@@ -3,15 +3,19 @@ package pl.touk.nussknacker.engine.avro.schemaregistry.confluent
 import java.nio.ByteBuffer
 
 import cats.data.Validated
+import io.confluent.kafka.schemaregistry.ParsedSchema
+import io.confluent.kafka.schemaregistry.avro.{AvroSchema, AvroSchemaProvider}
+import org.apache.avro.Schema
 import org.apache.kafka.common.errors.SerializationException
 
 object ConfluentUtils {
-  //Copied from AbstractKafkaAvroSerDe
+
+  private final val ValueSubjectPattern = "(.*)-value".r
```

Comment:
Can we just use `endsWith`, to remove regexp parsing/execution?

Reply:
It's not part of my change, but I agree with you: we can cache it. In fact we run it only at nk and once before deploy process.